pharmagkb.py

```diff
@@ -151,7 +161,8 @@ def generate_locations(elem, ns, chemical_nodes, pathway_id):
                match = go_location_re.match(location) 
                if match:
                    go_location = match.group(1)
-                   location_node = CConceptNode(go_location)
+                   go_type = find_gons.find_go_type(go_location)
```

Comment:
find_gons doesn't have find_go_type function

Reply:
sorry, I forgot to commit. I just did.